lib/screens/management_screen.dart

```diff
@@ -34,23 +38,27 @@ class _ManagementScreenState extends State<ManagementScreen> {
   /// Checks applications current and available version numbers
   Future _checkVersions() async {
     String currentVersion = await Updater.getCurrentVersion();
-    String serverVersion = (await Updater.checkVersion())
-        .elements
-        .firstWhere((element) =>
-            element.filters.first.value == configuration.getPlatform())
-        .versionName;
+    String? serverVersion = (await Updater.getServerVersion(
+      configuration.getPlatform(),
+    ));
 
     setState(() {
       _currentVersion = currentVersion;
-      _serverVersion = serverVersion;
+      _serverVersion = serverVersion ?? "Verkkovirhe";
       _loading = false;
+      _error = serverVersion == null;
```

Comment:
Error is rather unspecific term, maybe we should use something more qualified, e.g. serverVersionResolvingError

Reply:
Resolved.